lib/launcher.js

```diff
@@ -153,6 +154,26 @@ var init = function(configFile, additionalConfig) {
       }
     });
   }).then(function() {
+    // 3) If we're in `elementExplorer` mode, run only that.
+    if (config.elementExplorer) {
+      if (config.multiCapabilities.length != 1) {
+        throw new Error('Must specify only 1 browser while using elementExplorer');
```

Comment:
The steps above (1 and 2) resolve getMultiCapabilities/capabilities/multiCapabilities into just multiCapabilities. So someone can manually set the framework to `repl`/`explorer` and one of those capabilities config, but it should be fine as long as there is only 1 capability being resolved no?